### DISCUSSION TITLE
How do I set my public server to only block all plaintext DNS requests?

### DISCUSSION THREAD

**Discussion by o1x3** [AdguardTeam/AdGuardHome#6276] [Q&A] [answered]
Basically I have a publicly hosted Adguard Home server and it's been getting a lot of spam DNS requests from random South American countries. I can't block the IP because they're spamming from different IPs

![image](https://github.com/AdguardTeam/AdGuardHome/assets/34067026/db2ba621-88ee-4ff3-94cf-531c5d975700)

I've temporarily blocked access to apple.com, the only domain they've been pinging so far
I suppose the best thing to do is to close off the public port on server level and not allow DNS requests directly to the IP.

Is this possible?

**Comment by o1x3** [comment, marked as answer]
I blocked tcp/53 and udp/53 on the VPS. 
```
sudo iptables -A INPUT -p udp --dport 53 -j DROP

sudo iptables -A INPUT -p tcp --dport 53 -j DROP
```

**Comment by likkai** [comment]
i got this too and i guess you are hosting adguard home on oracle cloud? my solution is to simply allow only clients from certain ips, which include my home, school and mobile data's public ip address. that solves my problem

**Comment by snapsl** [comment]
How to become part of a DNS amplification attack.
